### DISCUSSION TITLE
How to access RejettoHFS3 on UNRAID with Nginx Proxy Manager without internal HTTPS cert files

### DISCUSSION THREAD

**Discussion by parhamsan** [rejetto/hfs#822] [Q&A]
Hey,

Thanks for releasing the RejettoHFS3 Beta on UNRAID community app.
I want to access RejettoHFS3 with my own subdomain (hfs3.mysubdomain.com) using nginx proxy manager.
I dont want to upload HTTPS Certificate files or keys; I want NPM to take care of the certificates.

I have assigned a static ip (192.168.1.123) to RejettoHFS3 container and set HFS_FORCE_HTTPS: false.

Also set HFS_HTTPS_PORT:9443 and HFS_PORT: 9000 on the UNRAID Docker.
I can confirm I can access the HFS with http://192.168.1.123:9000/ but when I try to access over https with NPM and my subdomain, I get

502 Bade Gateway
openresty

error

In the HFS options menu below are my settings:

![image](https://github.com/user-attachments/assets/4f2c31f6-aa1f-41d9-bb5a-44dfc8819ca7)

What am I doing wrong here?
All the other HFS instances on windows PCs are working with the same settings and NPM.

Any help would be appreciated!
Thanks again!!!

**Comment by rejetto** [comment]
i'm sorry, i don't really know Unraid. Bmartino1 is taking care of it, and I guess you can contact him at  
https://forums.unraid.net/topic/180463-support-rejetto-hfs-3/
or maybe at https://github.com/bmartino1/hfs-docker

**Reply by parhamsan** [reply]
Thanks for your reply,
Will post the question on UNRAID forums!

**Reply by bmartino1** [reply]
Working with user as well in unraid forum:
https://forums.unraid.net/topic/180463-support-rejetto-hfs-3/

Client may need to edit hfs docker port in Unraid advance hidden setting and potential fix other NPM configuration.

**Comment by rejetto** [comment]
i was very confused, because in the HFS world NPM is a part of node (node package manager), while you were talking about this Nginx Proxy Manager, apparently :)

**Comment by bmartino1** [comment]
I originally thought that too it took me a minute and a reread between the
2 Post.
